pages/common/git-svn.md

```diff
@@ -4,7 +4,11 @@
 
 - Clone an SVN repository:
 
-`git svn clone {{http://example.com/my_subversion_repo}} {{local_dir}}`
+`git svn clone {{http://example.com/subversion_repo}} {{local_dir}}`
+
+- Clone a SVN repository starting at a given revision number:
+
+`git svn clone -r{{1234}}:HEAD {{svn://svn.example.net/subversion/repo}} {{local_dir}}`
```

Comment:
It doesn't matter. SVN support both http and svn protocol. But better to use consistent url form.